run.sh

```diff
@@ -13,15 +20,206 @@ WP_DEBUG=${WP_DEBUG:-'false'}
   AFTER_URL=$(echo "$SEARCH_REPLACE" | cut -d ',' -f 2) || \
   SEARCH_REPLACE=false
 
-ERROR () {
-  echo -e "\n=> $(tput -T xterm setaf 1)$(tput -T xterm bold)ERROR$(tput -T xterm sgr 0) (Line $1): $2.";
-  exit 1;
+
+main() {
+
+  generate_config_for wp-cli
```

Comment:
I don't think this warrant a function. The function as a `if` statement and does 2 things. I prefer inlining the code rather than having wrapper function with obscure api and logic.